activerecord/test/cases/finder_test.rb

```diff
@@ -287,10 +287,8 @@ def test_exists_with_distinct_association_includes_limit_and_order
   end
 
   def test_exists_should_reference_correct_aliases_while_joining_tables_of_has_many_through_association
-    assert_nothing_raised do
-      developer = developers(:david)
-      developer.ratings.includes(comment: :post).where(posts: { id: 1 }).exists?
-    end
+    developer = developers(:david)
+    assert_equal false, developer.ratings.includes(comment: :post).where(posts: { id: 1 }).exists?
```

Comment:
@prathamesh-sonpatki can you change this to use `assert_predicate` instead? I think it reads better that way.

Reply:
Used `assert_not_predicate`.